internal/http/wait_for_n8n.go

```diff
@@ -0,0 +1,60 @@
+package http
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+	"task-runner-launcher/internal/logs"
+	"time"
+)
+
+func getRequest(url string) (*http.Response, error) {
+	client := &http.Client{
+		Timeout: 5 * time.Second,
+	}
+
+	req, err := http.NewRequest("GET", url, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	return client.Do(req)
+}
+
+func buildTaskRunnerServerHealthCheckURL(n8nURI string) string {
+	baseURL := n8nURI
+	if !strings.HasPrefix(n8nURI, "http://") && !strings.HasPrefix(n8nURI, "https://") {
+		baseURL = "http://" + n8nURI
+	}
```

Comment:
The `n8nURI` we pass is actually the task runner server's address (i.e. port 5679), which doesn't have the health endpoint. Either we need to pass another URI which has the n8n port, implement a separate health endpoint for the task runner server, or just GET the root URI and treat 404 as a successful response